ls8/cpu.h

```diff
@@ -5,14 +5,21 @@
 struct cpu {
   // TODO
   // PC
+  unsigned char pc;
   // registers (array)
+  unsigned char *registers;
```

Comment:
this totally works, but isn't as clear to the person reading the code as `unsigned char registers[8];`